bioinf-code-sources ChIPseq <!-- [![Chipseq]()](https://github.com/gwangjinkim/bioinf-code-sources) [![Build Status](https://travis-ci.org/danielecook/Awesome-Bioinformatics.svg?branch=master)]() -->
======================

Please feel free to [contribute](CONTRIBUTING.md)

## Full workflows

- [fastq.gz -> bwa (mm10, hg19; PE, SR) -> peaks (sh)](https://github.com/milospjanic/ChIPSeqFPro)
- [from download to peaks (sh, pl)](https://github.com/alfonsosaera/ChIPseq)
- [snakemake chipseq pipeline (R, py)](https://github.com/vrrenske/chipseq)

## Wrappers

- [python wrapper of MACS2](https://github.com/alfonsosaera/ChIPseq)

## Motif analysis

- [motif analysis scripts (R)](https://github.com/erinijapranckeviciene/chipseqt)

  chipseqt; some R functions for analysis of chipseq peaks, to determine transcription factor (TF) binding motifs and preferred colocation distances of pairs of TF motifs
  
## Peak analysis

- [Merging Peaks (R, Perl, bash programs)](https://github.com/marceladavila/MegePeaks)
- [Comparing differential transcription factor chip-seq methods](https://github.com/ying-w/chipseq-compare)

## Super enhancer analysis

- [process ChIPseq from fastq to peak calling and do Super Enhancer Search (R) (end to end)](https://github.com/belfordak/ChIPseq_SuperEnhancer_calling)

## Cut&Run

- [chipseq, cut&run pipeline](https://github.com/CompEpigen/ChIPseq_workflows)


## Miscellaneous Scripts

- [miscellaneous scripts (cpp)](https://github.com/haihantan/chipseq_tools)
